libs/domain/order/src/services/order.plugin.ts

```diff
@@ -0,0 +1,30 @@
+import { AuthService } from '@spryker-oryx/auth';
+import { App, AppPlugin } from '@spryker-oryx/core';
+import { resolve } from '@spryker-oryx/di';
+import { Subscription, tap } from 'rxjs';
+import { OrderService } from './order.service';
+
```

Comment:
Why do we need this plugin. Can we just clear storage on login?

Reply:
I wanted to keep it in the order package instead of the auth one.

Reply:
but you can check state auth in the order service. Not sure why we need plugin